MaterialDesignThemes.Wpf/ShadowAssist.cs

```diff
@@ -11,12 +11,27 @@ namespace MaterialDesignThemes.Wpf
 
     public enum ShadowDepth
     {
+        #region ObseleteDepth
         Depth0,
         Depth1,
         Depth2,
         Depth3,
         Depth4,
-        Depth5
+        Depth5,
+        #endregion
```

Comment:
I think actually putting the Obsolete attribute on these is a good idea. What are your thoughts on these suggested update values (since there are no exact equivalents)?

```c#
[Obsolete("Use Depth_0dp instead")]
Depth0,
[Obsolete("Considersing Depth_2dp instead")]
Depth1,
[Obsolete("Considersing Depth_3dp instead")]
Depth2,
[Obsolete("Considersing Depth_7dp instead")]
Depth3,
[Obsolete("Considersing Depth_12dp instead")]
Depth4,
[Obsolete("Considersing Depth_24dp instead")]
Depth5,
```

Reply:
That sounds good to me